From d3bdef36c2deaa1f781f6691f5995ff65899acd9 Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Thu, 24 Sep 2020 15:52:20 -0700
Subject: [PATCH 36/47] =?UTF-8?q?Yukon=E2=80=99s=20change=20is=20effective?=
 =?UTF-8?q?=202020-11-01.?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

* NEWS: Mention this.  (Thanks to Andrew G. Smith.)
* northamerica (America/Whitehorse, America/Dawson):
The rule change is Nov 1, not Mar 8 2:00.
---
 NEWS         | 13 +++++++++++++
 northamerica | 16 ++++++++++++++--
 2 files changed, 27 insertions(+), 2 deletions(-)

diff --git a/NEWS b/NEWS
index 6b9fd43..517d1c8 100644
--- a/NEWS
+++ b/NEWS
@@ -2,6 +2,10 @@ News for the tz database
 
 Unreleased, experimental changes
 
+  Briefly:
+    Revised predictions for Morocco's changes starting in 20203.
+    Canada's Yukon changes to -07 on 2020-11-01, not 2020-03-08.
+
   Changes to future timestamps
 
     Morocco's spring-forward after Ramadan is now predicted to occur
@@ -9,6 +13,15 @@ Unreleased, experimental changes
     (Thanks to Milamber.)  The first altered prediction is for 2023,
     now predicted to spring-forward on April 30 instead of April 23.
 
+  Changes to past and future time zone abbreviations and DST flags
+
+    Canada's Yukon, represented by America/Whitehorse and
+    America/Dawson, changes its time zone rules from -08/-07 to
+    permanent -07 on 2020-11-01, not on 2020-03-08 as 2020a had it.
+    This change affects only the time zone abbreviation (MST vs PDT)
+    and daylight saving flag for the period between the two dates.
+    (Thanks to Andrew G. Smith.)
+
   Changes to past timestamps
 
     Correct several transitions for Hungary for 1918/1983.
diff --git a/northamerica b/northamerica
index febf05b..1440c7a 100644
--- a/northamerica
+++ b/northamerica
@@ -2449,6 +2449,18 @@ Zone America/Creston	-7:46:04 -	LMT	1884
 # consistency with nearby Dawson Creek, Creston, and Fort Nelson.
 # https://yukon.ca/en/news/yukon-end-seasonal-time-change
 
+# From Andrew G. Smith (2020-09-24):
+# Yukon has completed its regulatory change to be on UTC -7 year-round....
+# http://www.gov.yk.ca/legislation/regs/oic2020_125.pdf
+# What we have done is re-defined Yukon Standard Time, as we are
+# authorized to do under section 33 of our Interpretation Act:
+# http://www.gov.yk.ca/legislation/acts/interpretation_c.pdf
+#
+# From Paul Eggert (2020-09-24):
+# tzdb uses the obsolete YST abbreviation for standard time in Yukon through
+# about 1970, and uses PST for standard time in Yukon since then.  Consistent
+# with that, use MST for -07, the new standard time in Yukon effective Nov. 1.
+
 # Rule	NAME	FROM	TO	TYPE	IN	ON	AT	SAVE	LETTER/S
 Rule	NT_YK	1918	only	-	Apr	14	2:00	1:00	D
 Rule	NT_YK	1918	only	-	Oct	27	2:00	0	S
@@ -2503,12 +2515,12 @@ Zone America/Inuvik	0	-	-00	1953 # Inuvik founded
 Zone America/Whitehorse	-9:00:12 -	LMT	1900 Aug 20
 			-9:00	NT_YK	Y%sT	1967 May 28  0:00
 			-8:00	NT_YK	P%sT	1980
-			-8:00	Canada	P%sT	2020 Mar  8  2:00
+			-8:00	Canada	P%sT	2020 Nov  1
 			-7:00	-	MST
 Zone America/Dawson	-9:17:40 -	LMT	1900 Aug 20
 			-9:00	NT_YK	Y%sT	1973 Oct 28  0:00
 			-8:00	NT_YK	P%sT	1980
-			-8:00	Canada	P%sT	2020 Mar  8  2:00
+			-8:00	Canada	P%sT	2020 Nov  1
 			-7:00	-	MST
 
 
-- 
1.8.3.1

